src/test/java/github/grace000/erversay/RequestParserTest.java

```diff
@@ -0,0 +1,22 @@
+package github.grace000.erversay;
+
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class RequestParserTest {
+    private String request = "GET /simple_get HTTP/1.1";
+    private RequestParser parser;
+
+    @Test
+    public void itReturnsMethodPathVersionRequestArray() {
+        parser = new RequestParser();
+
+        String[] parsedRequest = parser.parse(request);
```

Comment:
Are the elements in the `String[]` always the same thing in the same order? If so, can't you come up with a representation that's easier to work with? Hard-coding array indices throughout your code is hard to read and will probably become tedious to write and maintain.